src/requestor/routes/query.py

```diff
@@ -96,11 +96,15 @@ async def list_user_requests(api_request: Request, auth=Depends(Auth)) -> dict:
 
     Use the "active" query parameter to get only those requests
     created by the user that are not in DRAFT or FINAL statuses.
+
+    Add filter values as key=value pairs in the query string
+    to get filtered results.
```

Comment:
could you copy+update the long explanation with all the details (or, and, etc) and examples (linked in the design doc)?